elements/urth-viz-table/urth-viz-table.html

```diff
@@ -278,6 +278,7 @@
 
                     var tableContainer = this.$.tableContainer;
                     tableContainer.innerHTML = '';
+                    tableContainer.style.height = 'auto';
```

Comment:
@lbustelo Difference is that if you have a big table (the height is big) and then the datarows shrink, the height is already set a big number, no longer 'auto', so even though _updateDatarows calls _createTable to recreate the table, the div for the table still keeps the big height, which we need to reset back to auto.